os/os.proto

```diff
@@ -321,6 +370,9 @@ message VerifyResponse {
   string activation_fail_message = 2;
 
   VerifyStandby verify_standby = 3;
+  // Targets that require the Install/Activate/Verify process executed once
+  // per supervisor reply with individual_supervisor_install set to true
+  bool individual_supervisor_install = 4;
```

Comment:
Dual Supervisor Targets that require the Install/Activate/Verify process executed once per supervisor reply with individual_supervisor_install set to true